common/src/test/scala/explore/model/arb/ArbExploreLocalPreferences.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model.arb
+
+import explore.model.ExploreLocalPreferences
+import org.scalacheck.Arbitrary
+import org.scalacheck.Cogen
+import org.scalacheck.Cogen._
+import org.scalacheck.Gen
+import typings.loglevel.mod.LogLevelDesc
+
+trait ArbExploreLocalPreferences {
+
+  implicit val localPreferencesArb = Arbitrary[ExploreLocalPreferences] {
+    for {
+      level <- Gen.const(LogLevelDesc.INFO)
```

Comment:
Do we want to generate the other values? Actually if we make an `Enumerated` we can use its functionality.